llvm/lib/TableGen/Record.cpp

```diff
@@ -3102,6 +3102,21 @@ Record::getValueAsListOfDefs(StringRef FieldName) const {
   return Defs;
 }
 
+llvm::IRMemLocation Record::getLocationTypeAsInt(StringRef FieldName) const {
```

Comment:
This function does not belong here. It should be moved instead to the specific backend that uses it (intrinsic emitter I am assuming)

Reply:
Hello @jurahul 
Do you mind expanding what you would like to see?
Because this function is called  by CodeGenIntrinsic::setProperty(const Record *R)  in CodeGenIntrinsics.cpp, in there , AFAIK I cannot check the target architecture. So I would like some pointer to know how I should add this in the backend  of the target specific architecture.

Reply:
Thank you for clarifying.
I believe I've addressed your comment in the latest update.